lib/rules/declaration-property-value-keyword-no-deprecated/index.mjs

```diff
@@ -161,7 +158,6 @@ const rule = (primary, secondaryOptions) => {
 
 				report({
 					message,
-					messageArgs,
```

Comment:
[question] Why doesn't this rule use `messageArgs`?

Reply:
Understood. But if we need to consider the differentiation, we should keep this rule's `messageArgs` as-is in this PR.

In addition, the "Experimental feature" expression should also be kept because this feature is still unstable. For now, we cannot ensure the feature stability.

Reply:
> the "Experimental feature" expression should also be kept because this feature is still unstable

OK.